webpack.config.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable global-require */
```

Comment:
- [ ] Please remove this eslint disable line its best to fix the issue in the code.